README.md

```diff
@@ -37,7 +37,7 @@ install.packages("EGRET")
     ## package 'EGRET' successfully unpacked and MD5 sums checked
     ## 
     ## The downloaded binary packages are in
-    ##  C:\Users\lcarr\AppData\Local\Temp\1\RtmpAPk7Z0\downloaded_packages
+    ##  C:\Users\lcarr\AppData\Local\Temp\1\Rtmp8QoZAB\downloaded_packages
```

Comment:
So keep `messages=TRUE` for code chunks is what you are thinking? It's nice, but does make it look like my changes impacted the README when they really didn't.